### PR TITLE
new lab introducing fields, class instantiation, object state

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,30 +2,163 @@
...
+- Use IntelliJ's Java Visualizer to view object state
+- Assign an instance variable an initial value at variable declaration
+- Use IntelliJ to auto-generate a `toString()` method for printing object state
+- Use dot notation to access and update object state
 
-## Introduction
+## Instructions
 
-A 1-2 sentence summary of what will be covered.
+The starter project contains a class named `Book` with a `main` method that contains comments outline the lab tasks.
```
Comment (kcleland0818) [inline README.md]: "The starter project contains a class named `Book` with a `main` method that contains comments outline the lab tasks."

Should "outline" be "outlining"?

File: README.md [hunk 2]
```diff
@@ -2,30 +2,163 @@
...
+or end of program:
+
+![continue execution](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-oop-fundamentals/resume_program.png)
+
+## Task #1 - Add fields and create 3 `Book` objects
+
+1. Edit the `Book` class to add 5 fields named `title`, `isbn`, `edition`, `pages`, and `language`.
+   The title, isbn, and language are strings.  The edition and pages are integers.
+2. Edit the `main()` method to create 3 instances of the `Book` class.  Declare local variables named
+   `book1`, `book2`, and `book3` and assign each variable to reference a new `Book` instances.
```
Comment (kcleland0818) [inline README.md]: "... and assign each variable to reference a new `Book` instances."

I believe "instances" should be singular: "instance".

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md]: fixed

**Review by kcleland0818 — APPROVED** [review]

**Review by developerjaym — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**